src/Microsoft.Identity.Web/Resource/RolesOrScopesRequiredHttpContextExtensions.cs

```diff
@@ -0,0 +1,94 @@
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Security.Claims;
+using global::Microsoft.AspNetCore.Http;
+using global::Microsoft.Identity.Web;
+using Bludata.Blu.Web.Security.Entities;
+using Bludata.Blu.Web.Security.Interface;
+
+namespace Bludata.Blu.Web.Security.Utils;
```

Comment:
namespace needs to be updated and using removed.
please add license info like we have on other classes

Reply:
I did it, thanks for letting me know. Hope it will be useful.